src/join.sol

```diff
@@ -117,6 +140,13 @@ contract DaiJoin {
     DSTokenLike public dai;  // Stablecoin Token
     uint    public live;     // Active Flag
 
+    // Events
+    event Rely(address indexed usr);
+    event Deny(address indexed usr);
+    event Join(address indexed urn, uint256 wad);
+    event Exit(address indexed guy, uint256 wad);
```

Comment:
Same here. I know this might be a bit confusing wit the `usr` of `Rely` and `Deny` but I think I still prefer to keep the same naming than the functions.

Reply:
Yeah, this is a bigger issue where the different join adapters use inconsistent variable names. I was going to go through in a separate PR and clean all of this up but I'll just do it here.